### ISSUE TITLE
Ucto sentence splitting can cause FoLiA text redundancy errors

### ISSUE THREAD

**Issue by proycon** [LanguageMachines/ucto#50]
This is a pretty tough one I'm afraid. Take the following FoLiA paragraph from a real example:

```xml
<p xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1" class="p">                                                                                                                                                                                  
 <t>Zoo ooit, dan heb ik in de laatste maanden betreurd dat ik mijn ‘dagboek’ zoo geheel en al verwaarloosd heb. ’t Gaat mooi met de ‘Katholieke Sociale Actie’ (<t-style class="italic">K.S.A.</t-style>). Wat zou ’t me nu niet waard zijn in dit dagboek eens een precies verhaal te bezitten, hoe die zaak is tot stand gekomen.</t>
</p>
```

Ucto tokenises this wrongly as it inserts a sentence split inside the K.S.A abbreviation:

```xml
              <w xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1.s.2.w.12" class="WORD" set="tokconfig-nld" space="no">                                                                                                                                          
                <t>K</t>                                                                                                                                                                                                                                                                                                                                                                                                                                                               
              </w>                                                                                                                                                                                                                                             
              <w xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1.s.2.w.13" class="PUNCTUATION" set="tokconfig-nld" space="no">                                                                                                                                   
                <t>.</t>                                                                                                                                                                                                                                                                                                                                                                                                                                                               
              </w>                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
            </s>                                                                                                                                                                                                                                               
            <s xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1.s.3">                                                                                                                                                                                             
              <w xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1.s.3.w.1" class="WORD" set="tokconfig-nld" space="no">                                                                                                                                           
                <t>S</t>                                                                                                                                                                                                                                                                                                                                                                                                                                                           
              </w>                                                                                                                                                                                                                                             
              <w xml:id="TEI.1.text.1.body.1.div1.1.div2.1.p.1.s.3.w.2" class="PUNCTUATION" set="tokconfig-nld" space="no">                                                                                                                                    
                <t>.</t>                                                                                                                                                                                                                                       
                <pos class="LET()" confidence="1" head="LET" />                                                                                                                                                                                                
                <lemma class="." />                                                                                                                                                                                                                            
              </w>                                                                                                                                                                                                                                             
            </s>                                                                                                                                                                                                                                               
```

Of course wrong tokenisation can and will occur (and though not ideal that is not the issue of this actual bug report), however, the problem now is that this produces a text consistency error, so ucto/frog generates invalid FoLiA! Because of the sentence split we get "K.\sS.A" instead of "K.S.A" and FoLiA v1.5 trips over this, which is pretty serious.

**Comment by proycon** [comment]
The above was detected with ``foliavalidator`` by the way, it seems ``folialint`` does not complain here. I'll create a minimal example document..

**Comment by proycon** [comment]
Minimal example attached. Strangely enough, the tokenisation problem reproduces but the validation errors do not (which is good, that might mean the validators are lenient enough to handle this corner case). But that doesn't explain yet why I did get the error earlier..

input: https://download.anaproy.nl/uctoissue50.xml
output: https://download.anaproy.nl/uctoissue50.out.xml

**Comment by proycon** [comment]
The entire input document was pretokenised with ucto by INT and those validate fine... 
Only the Frog result for some reason doesn't! Still digging....

**Comment by proycon** [comment]
Ok, this is a bug in the python folia library, not an ucto nor frog issue, continuing here: proycon/pynlpl#41 

The abbreviation problem is addressed separately in #51
